cg/apps/housekeeper/hk.py

```diff
@@ -390,6 +394,24 @@ def is_fastq_or_spring_in_all_bundles(self, bundle_names: List[str]) -> bool:
             )
         return all(sequencing_files_in_hk.values())
 
+    def get_non_archived_files(self, bundle_name: str, tags: Optional[list] = None) -> List[File]:
+        """Returns all files from given bundle, with given tag, which have not been archived."""
+        return self._store.get_non_archived_files(bundle_name=bundle_name, tags=tags)
+
+    def get_archived_files(self, bundle_name: str, tags: Optional[list] = None) -> List[Path]:
+        """Returns all files from given bundle, with given tag, which have been archived."""
+        return self._store.get_archived_files(bundle_name=bundle_name, tags=tags)
+
+    def add_archives(self, files: List[Path], archive_task_id: int) -> None:
```

Comment:
```suggestion
    def add_file_archives(self, files: List[Path], archive_task_id: int) -> None:
```

Reply:
I find `add_archives` clearer

Reply:
`add_archives` would run into problems if there are other types of archives in the future as it is non explicit.